addons/api/addon/serializers/account.js

```diff
@@ -33,7 +33,7 @@ export default class AccountSerializer extends ApplicationSerializer {
    * `attributes` entirely on update.
    */
   serializeOIDC(snapshot) {
-    const { isNew } = snapshot?.record;
+    const { isNew } = snapshot?.record || null;
```

Comment:
what's the reason behind adding this `null` value here?  Did it error out during this update?

Reply:
@DhariniJeeva
This error come with the update, as a side effect of this update also updating the `eslint-ember-plugin` and the recommended rules for JS.

The exact rules that affects this is `no-unsafe-optional-chaining`, more [info about here](https://eslint.org/docs/rules/no-unsafe-optional-chaining).

In a nutshell, if `snapshot.record` does not exist, `isNew` will not be destructured and it will have a `undefined` value. To avoid that I thought best will be setting the value to null as fallback.
But happy to discuss other options here :smiley:

Reply:
```suggestion
    const { isNew } = snapshot?.record || {};
```